packages/moonstone/internal/Picker/Picker.js

```diff
@@ -866,11 +851,10 @@ const PickerBase = class extends React.Component {
 				ref={this.initContainerRef}
 			>
 				<PickerButton
+					{...voiceProps}
```

Comment:
Same question here on applying the same voice props to multiple components.

Reply:
This component is also fixed. Thanks~